README.md

```diff
@@ -1,72 +1,84 @@
-<a href="https://unwrapped.dev"><img src="https://github.com/middlewarehq/unwrapped/assets/7949047/b8a01018-e5dd-4d0f-a5da-ab3cfe039862" width="100%" /></a>
+
+[![Unwrapped Logo](https://github.com/middlewarehq/unwrapped/assets/7949047/b8a01018-e5dd-4d0f-a5da-ab3cfe039862)](https://unwrapped.dev)
 
 <blockquote align="center">
   <h3>
     <i>
-      Spotify Unwrapped - for devs
+      Dive into Unwrapped - for coders
       <br/>
-      Showcase your contributions just like you flex your music!
+      Reveal your coding journey just as you showcase your musical taste!
       <br />
-      Check it out at https://unwrapped.dev
+      Explore more at  https://unwrapped.dev
     </i>
   </h3>
 </blockquote>
 
+
+
+# Unwrapped - by Middleware :      <a href="https://unwrapped.dev"><img src="https://github.com/devdnv/wrapped/assets/147969535/c9e6dfc5-4e55-4c2b-b09b-8d4c405378a8" width="5%" align='right' /></a>
+
+
+
+
+![Stargazers](https://img.shields.io/github/stars/middlewarehq/unwrapped?style=social) 
+![Forks](https://img.shields.io/github/forks/middlewarehq/unwrapped?style=social) 
+![Contributors](https://img.shields.io/github/contributors/middlewarehq/unwrapped?color=dark-green) 
+![Issues](https://img.shields.io/github/issues/middlewarehq/unwrapped) 
+![License](https://img.shields.io/github/license/middlewarehq/unwrapped) 
+
+
+
+Welcome to **Unwrapped - by Middleware**! This application is designed to provide software developers with insights into their coding activities throughout the year, much like Spotify Wrapped for developers.
+
 ---
 
-<a href="https://unwrapped.dev"><img src="https://github.com/middlewarehq/year-in-review/assets/7949047/8984725a-dcd0-4568-9922-9e50db20313f" width="100%" /></a>
-
-## Welcome to **Unwrapped - by Middleware** :technologist:!  
-This app is designed to provide you with insights into your coding activities throughout the year, think Spotify  Wrapped but for software developers.  
-
-<h2 align='center'>🚀 Features 🚀</h2>
-
-- **Code Contributions :medal_military:**: View a summary of your contributions to various repositories over the past year. See where you stand !
-- **Productivity Insights :woman_technologist:**: Discover how long you spent waiting for reviews, and who helped you out the most with code reviews.
-- **Repo Highlights**: Highlight noteworthy projects you worked on during the year.
-- **Coding Habits :owl:**: Gain insights into your coding habits, such as peak coding hours and most active days.
-- **Cobtribtuion Styles :evergreen_tree:**: Are you pushing your code, or are you stuck in Pull Request Reviews ?
-- **Catching fires before they spread :firefighter:**: Are you often requesting changes and finding bugs before they reach production ?
-- **And Much More :woman_astronaut:**
-
-<a href="https://unwrapped.dev"><img src="https://github.com/middlewarehq/unwrapped/assets/7949047/390e9d7a-183e-41c6-bcde-599d91df732f" width="100%" /></a>
-
```

Comment:
I think undo the image changes between features and project setup section.
Super appreciate you taking a stab at it.

Here's the reasons behind why it was the way it was:
1. Limit the number of images so it doesn't need too much scrolling to get to the main thing.
2. The height of the existing banner was chosen specifically so a few more names of the popular devs show up in it. The reduced height further crops those out.